docs/json-api.md

```diff
@@ -335,11 +335,30 @@ const styles = {
 Compiles to:
 
 ```css
-.button-jss-0-1 {
+.button-0-1 {
   color: '#0000B3'
 }
 ```
 
+## Alternative syntax using string templates
+
+You can use a template string literal or a regular string to declare the props and values.
+This notation is only supported for style declarations, nested rules, media queries, keyframes and others still have to be objects.
+
```

Comment:
so with tag function like `css` it can be something like

```js
button: css`
  padding: 10px;
  color: ${theme => theme.primary ? 'red' : 'black'}
`
```

this is also an approach for static optimizations via babel